### PR TITLE
MJPEG Compression

### PR DESCRIPTION
**Pull request by jochem725**
There is now some form of compression involved in distributing streams.
The compressions works by resizing the image, greatly reducing the amount of data sent to the client.

It might be heavy for some computers, thats why I've added a toggle to the config, so you can turn it off if your computer is melting apart.

### CODE REVIEW

**Comment by Kingdorian** [comment]
Looks really good.

File: backend/src/main/java/com/benine/backend/video/MJPEGFrameHeader.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  private String contenttype;
+
+  /**
+   * Creates a frameheader object from a bytestream header.
+   * @param header Header in bytes.
+   */
+  public MJPEGFrameHeader(byte[] header) {
+    String headerString = new String(header, StandardCharsets.UTF_8);
+
+    this.boundary = "--polycast";
```
Comment (Kingdorian) [inline backend/src/main/java/com/benine/backend/video/MJPEGFrameHeader.java]: I don't think we should use hardcoding like this.

Comment (jochem725) [inline backend/src/main/java/com/benine/backend/video/MJPEGFrameHeader.java]: I've fixed this by creating final variables.

**Comment by egedebruin** [comment]
Very good clean code and good enhancement, I agree with merging.

**Comment by bvanwijk** [comment]
I think this can be merged.

**Pull request merged**